tests/e2e/pageObjects/browser-page.ts

```diff
@@ -282,6 +304,24 @@ export class BrowserPage {
         await t.click(this.addKeyButton);
     }
 
+    /**
+     * Adding a new Stream key
+     * @param keyName The name of the key
+     * @param TTL The Time to live value of the key
+     * @param field The field name of the key
+     * @param value The value of the key
```

Comment:
It is not in the same order as the function

Reply:
fixed